File: Assignment_11/question5.c
///////////////////////////////////////////////////////////////////////////////////
//
//  File Name   : question5.c
//  Description : Display even numbers pattern.
//
//  Problem     : Accept number from user and display pattern: 2 4 6 8 10 12 14 16
//
//  Author      : Asawari Jadhav
//  Date        : 20/05/2025
//
///////////////////////////////////////////////////////////////////////////////////

#include <stdio.h>

void Pattern(int iNo)
{
  int iCnt = 0;
  for (iCnt = 1; iCnt <= iNo; iCnt++)
  {
    printf("%d\t", iCnt * 2);
  }
}

int main()
{
  int iValue = 0;

  printf("Enter number of elements\n");
  scanf("%d", &iValue);

  Pattern(iValue);

  return 0;
}

File: Assignment_50/question5.java
//////////////////////////////////////////////////////////////////////////////////////////////
//
//  File Name     : question5.java
//  Description   : Accept number of rows and columns from user and display number pattern
//  Problem       : Display numbers on borders and diagonals, spaces elsewhere
//
//  Author        : Asawari Jadhav
//
//  Date          : 21-08-2025            
//
//////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Pattern 
{
    public void DisplayPattern(int iRow, int iCol) 
    {
        for (int i = 1; i <= iRow; i++) 
        {
            for (int j = 1; j <= iCol + 1; j++) 
            {
                if (i == 1 || i == iRow || j == 1 || j == iCol + 1 || i == j)
                {
                    System.out.print(j + " ");
                }
                    
                else
                {
                    System.out.print("  ");
                }
                    
            }
            System.out.println();
        }
    }
}

public class question5 
{
    public static void main(String args[]) 
    {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter number of rows: ");
        int iRow = sc.nextInt();
        System.out.print("Enter number of columns: ");
        int iCol = sc.nextInt();

        Pattern pobj = new Pattern();
        pobj.DisplayPattern(iRow, iCol);

        
    }
}
